app/templates/public.hbs

```diff
@@ -7,10 +7,10 @@
         <div class="ui info d-flex">
           {{#if this.model.logoUrl}}
             <LinkTo @route="public">
-              <img src="{{this.model.logoUrl}}" class="logo mr-8" />
+              <img src="{{this.model.logoUrl}}" class="logo mr-8" style="margin-bottom: -120px;"/>
```

Comment:
I added inline CSS here coz I couldn't create just another class for the logo as there is one already and plus I had to do changes in margin-bottom and not margin-right